README.md

```diff
@@ -737,7 +737,7 @@ const Example = () => (
           actions.setSubmitting(false)
         }, 1000);
       }}
-      render={(props: FormikProps<Values>) =>
+      render={props =>
```

Comment:
Why did `FormikProps<Values>` go away here?

Reply:
Here, types are inferred. No need to specify them ;)

Reply:
Yes, but they would be inferred as `FormikProps<any>`, unless I am mistaken (which i could be)

Reply:
Indeed. But in this case we don't have `Values` type. And the `render` method doesn't use `props.values`. So I thought this explicit type was useless.